src/web/js/trove/chart-lib.js

```diff
@@ -1418,6 +1430,14 @@
       options['trendlines'][0]['degree'] = trendlineDegree;
     }
 
+    if (dotChartP) {
+      options['vAxis'] = {
+        ticks: [],
+        minValue: 0,
+        maxValue: 10,
```

Comment:
We should comment this.

Reply:
@ds26gte your comment still doesn't address the important bit, which is the part you told me you "never would have guessed" yesterday: google charts needs some form of maxValue, otherwise having all the dots at y=0 will force the vAxis to be _centered_ at 0. 

If you never would have guessed this, I suspect it would be worth documenting in the comment!

Reply:
Done

Reply:
Done